core/src/com/unciv/logic/automation/civilization/NextTurnAutomation.kt

```diff
@@ -857,7 +859,7 @@ object NextTurnAutomation {
         // Don't accept if they are at war with our friends, they might use our land to attack them
         if (civInfo.diplomacy.values.any { it.isRelationshipLevelGE(RelationshipLevel.Friend) && it.otherCiv().isAtWarWith(otherCiv)})
             return false
-        if (hasAtLeastMotivationToAttack(civInfo, otherCiv, civInfo.getDiplomacyManager(otherCiv).opinionOfOtherCiv().toInt()) > 0)
+        if (hasAtLeastMotivationToAttack(civInfo, otherCiv, (civInfo.getDiplomacyManager(otherCiv).opinionOfOtherCiv()/ 2 - 10).toInt()) >= 0)
```

Comment:
How about moving the 10 to the right side of the equation, so it's clear which part is configurable?

Reply:
That sounds good on paper. But opinionOfOtherCiv shortcuts if it is guaranteed to be below the atLeast value. Therefore it would return early and not have the effect that I want.